packages/paste-website/src/pages/patterns/confirmations/index.mdx

```diff
@@ -0,0 +1,190 @@
+
+### Composition notes
+
+Each Confirmation [Alert Dialog](/components/alert-dialog) should follow a predictable content pattern. 
+
+#### Heading
+
+The [Heading](/components/heading) should be phrased as a question with two distinct options. In general:
+- Avoid using articles like “the” or “a.” 
+- Avoid the phrase “Are you sure?” 
+- Use sentence case.
+
+#### Paragraph (optional)
+
+The [Paragraph](/components/paragraph) should include 1-3 sentences that explain the outcomes of the actions. If the action is irreversible, say so.
+
+#### Buttons
+Confirmation messages typically have 2 (Buttons)[/components/button]: one that confirms the action and another that offers an alternative. Labels should be short and mirror the language in the title whenever possible.
+
+## Variations
+
+Confirmation messages are used in a variety of situations where it’s important to confirm that the user wants to proceed with an action, like saving changes before exiting, starting a process that could incur unexpected costs, or [deleting something](/patterns/delete).
+
+<Box marginBottom="space70">
+  <Table scrollHorizontally>
+    <THead>
+      <Tr>
+        <Th width="200px">Use case</Th>
+        <Th>Scenario</Th>
+        <Th width="200px">Sample text</Th>
+      </Tr>
+    </THead>
+    <TBody>
+      <Tr verticalAlign="top">
+        <Td> User wants to [delete](/patterns/delete) something </Td>
+        <Td> **Delete [thing]?** <br/> You’re deleting “[object name]”. You cannot undo this action. <br/> Cancel | Delete [thing] </Td>
+      </Tr>
+      <Tr verticalAlign="top">
+        <Td> User wants to [delete](/patterns/delete) something that has major consequences</Td>
+        <Td> **Delete [thing]?** <br/> When you delete [thing], all [objects] are permanently deleted. You cannot undo this action. <br/> Enter [user input] to delete [thing] <br/> Cancel | Delete [thing] </Td>
+      </Tr>
+      <Tr verticalAlign="top">
+        <Td> User wants to change something</Td>
+        <Td> **Change [thing]?** <br/> When you change [thing], your [objects] are deleted. <br/> Cancel | Change [thing] </Td>
+      </Tr>
+      <Tr verticalAlign="top">
+        <Td> User wants to leave a flow but may want to save changes</Td>
+        <Td> **Save changes?** <br/> If you close without saving, changes are lost. <br/> Close without saving | Save and close </Td>
+      </Tr>
+      <Tr verticalAlign="top">
+        <Td> User wants to leave a flow but changes can’t be saved</Td>
+        <Td> **Stop [doing thing]?** <br/> Your changes will not be saved. <br/> Keep [doing thing] | Stop [doing thing] </Td>
+      </Tr>
+      <Tr verticalAlign="top">
+        <Td> The customer needs to swap regions to access the data.</Td>
+        <Td> **Use title to repeat action** <br/> Explain what could happen to the resource, and offer other options if available and appropriate.</Td>
+      </Tr>
+    </TBody>
+  </Table>
+</Box>
```

Comment:
Honestly, tables are much easier to do with pure Markdown syntax. https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables. It's very rare that we actually need to do Component pages tables

Reply:
Changed it!